### PR TITLE
[en-lang] Replace shortcode code with codenew

### PR DESCRIPTION
**Pull request by DanyC97**
Moving to `codenew` shortcode so we can remove `code.html` after we tackle `zn` too

### CODE REVIEW

**Comment by DanyC97** [comment]
@tengqm fyi , thank you

File: content/en/docs/tasks/federation/set-up-placement-policies-federation.md [hunk 1]
```diff
@@ -86,15 +86,15 @@ Create a Service in the host cluster to contact the external policy engine:
...
+{{< codenewnew file="federation/policy-engine-service.yaml" >}}
 
 Create a Deployment in the host cluster with the Federation control plane:
 
     kubectl create -f policy-engine-deployment.yaml
 
 Shown below is an example Deployment for OPA.
 
-{{< codenew file="federation/policy-engine-deployment.yaml" >}}
+{{< codenewnew file="federation/policy-engine-deployment.yaml" >}}
```
Comment (tengqm) [inline content/en/docs/tasks/federation/set-up-placement-policies-federation.md]: ...

File: content/en/docs/tasks/federation/set-up-placement-policies-federation.md [hunk 2]
```diff
@@ -86,15 +86,15 @@ Create a Service in the host cluster to contact the external policy engine:
 
 Shown below is an example Service for OPA.
 
-{{< codenew file="federation/policy-engine-service.yaml" >}}
+{{< codenewnew file="federation/policy-engine-service.yaml" >}}
```
Comment (tengqm) [inline content/en/docs/tasks/federation/set-up-placement-policies-federation.md]: error

File: content/en/docs/tasks/federation/set-up-placement-policies-federation.md [hunk 3]
```diff
@@ -36,7 +36,7 @@ received from the external policy engine.
 
 Shown below is an example ConfigMap for the Admission Controller:
 
-{{< codenew file="federation/scheduling-policy-admission.yaml" >}}
+{{< codenewnew file="federation/scheduling-policy-admission.yaml" >}}
```
Comment (tengqm) [inline content/en/docs/tasks/federation/set-up-placement-policies-federation.md]: error

File: content/en/docs/tasks/federation/set-up-placement-policies-federation.md [hunk 4]
```diff
@@ -130,7 +130,7 @@ Annotate one of the clusters to indicate that it is PCI certified.
 
 Deploy a Federated ReplicaSet to test the placement policy.
 
-{{< codenew file="federation/replicaset-example-policy.yaml" >}}
+{{< codenewnew file="federation/replicaset-example-policy.yaml" >}}
```
Comment (tengqm) [inline content/en/docs/tasks/federation/set-up-placement-policies-federation.md]: error

File: content/en/docs/tasks/federation/set-up-placement-policies-federation.md (hunk 1 above)
Comment (DanyC97) [reply content/en/docs/tasks/federation/set-up-placement-policies-federation.md]: sorry for all this, i'll fix it asap

**Comment by DanyC97** [comment]
@tengqm please help review, thx

**Comment by DanyC97** [comment]
do i need to include the changes i made for `zh` lang into this PR so it gets merged into master branch ?

let me know and i'll do so

**Comment by tengqm** [comment]
@DanyC97 Pls avoid mixing contents for different languages in the same PR because we have different groups of reviewers for each language. Regarding this PR, the `policy.rego` file is not an example yaml/json file, we should avoid using the `codenew` shortcode because the shortcode is not designed for this purpose.

**Comment by DanyC97** [comment]
understood, will update / take out the file shortly

**Comment by DanyC97** [comment]
how does it look now @tengqm ?

On a different note, following your last comment does that mean that we will never be in a position to drop `code.html` because of `policy.rego` ? trying to get a clear picture hence my question

**Comment by tengqm** [comment]
Having the "policy.rego" file blocking the removal of `code.html` is annoying. We may probably want to embed the content of that "rego" file into the markdown file, as a workaround.  If you check the rego file, you will find it is mostly comment (documentation). It is not a YAML/JSON file to be checked using the examples_test utility we have at CI gate. It doesn't look like a file that will be tried out for most users.

**Comment by tengqm** [comment]
@DanyC97 Since the change to 'policy.rego' will warrant a separate PR. I'm kicking this in. Thanks for the great effort.

/lgtm
/approve

**Comment by DanyC97** [comment]
@tengqm thank you for helping out, much appreciated.
I'll take a look at the `policy.rego` file in another PR soon

**Pull request merged**